packages/bullet-list-react/src/Li.tsx

```diff
@@ -0,0 +1,10 @@
+import React, { ReactNode } from "react";
+
+interface Props {
+    children: ReactNode;
+    nested?: boolean;
```

Comment:
See comment below for proposed change. We want to remain semantically correct when nesting list elements.

Reply:
See PR #95 for proposed implementation